readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -0,0 +1,186 @@
+/* Read the Docs - Documentation promotions */
+
+var constants = require('./constants'),
+    rtddata = require('./rtd-data');
+
+var rtd;
+
+function init() {
+    var post_data = {},
+        force,
+        params;
+
+    rtd = rtddata.get();
+
+    if (!rtd.show_promo()) {
+        return;
+    }
+
+    post_data.placements = get_placements(rtd);
+    post_data.project = rtd.project;
+
+    if (typeof URL !== 'undefined' && typeof URLSearchParams !== 'undefined') {
+        // Force a specific promo to be displayed
+        params = new URL(window.location).searchParams;
+        force = params.get('promo');
```

Comment:
Could use a better variable name here, this reads like a boolean. Perhaps `explicit_promo` or `forced_promo`

Reply:
Sounds good.